test-loop-tests/src/tests/congestion_control.rs

```diff
@@ -59,7 +65,55 @@ fn slow_test_congestion_control_simple() {
         .shutdown_and_drain_remaining_events(Duration::seconds(20));
 }
 
-fn setup(accounts: &Vec<AccountId>) -> (TestLoopEnv, AccountId) {
+#[cfg_attr(not(feature = "test_features"), ignore)]
+#[test]
+fn slow_test_one_shard_congested() {
+    init_test_logger();
+
+    let mut accounts = make_accounts(NUM_PRODUCERS + NUM_VALIDATORS);
+    let shard1_acc1: AccountId = "0000".parse().unwrap();
+    let shard1_acc2: AccountId = "1111".parse().unwrap();
+    let shard2_acc1: AccountId = "xxxx".parse().unwrap();
```

Comment:
Can you add assertions that the accounts are indeed in the shards you want them to be in? Or just that they are in different shards.